### PR TITLE
Feature add query strings

### PR DESCRIPTION
**Pull request by mrockingham**
# Description
Feature Add Query Strings
Fixes # (issue)

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Change Status
- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?
- [ ] Test A
- [ ] Test B

# Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: api/incidents/incidentsRouter.js [hunk 1]
```diff
@@ -107,6 +107,40 @@ const { dsFetch } = require('../dsService/dsUtil');
...
+      incident.categories = JSON.parse(incident.categories);
+      incident.date = dateObj.getMonth();
+      return incident;
+    });
+
+    let filterResponse = queryResponse.filter((item) => {
+      let searchStartDate = +req.query.startDate;
+      let searchEndDate = +req.query.endDate;
+
+      if (req.query.state) {
```
Comment (jstewart8053) [inline api/incidents/incidentsRouter.js:127]: Maybe add some comments here to clarify exactly what is being searched. It looks like it is only being searched by the month. And also what happens if the user selects the same month as the start and end date?

File: INSTALLATION_INSTRUCTIONS.md [hunk 2]
```diff
@@ -55,8 +55,8 @@ See .env.sample for example values
 
 There are 3 options to get postgresql installed locally [Choose one]:
 
-1. Use docker. [Install](https://docs.docker.com/get-docker/) for your platform
-    - run: `docker-compose up -d` to start up the postgresql database and pgadmin.
+1. Use docker. [Install](https://docs.dock  er.com/get-docker/) for your platform
```
Comment (jlong5795) [inline INSTALLATION_INSTRUCTIONS.md:58]: Is there a reason we modified this reference instruction?

File: api/incidents/incidentsRouter.js (hunk 1 above)
Comment (jlong5795) [reply api/incidents/incidentsRouter.js:127]: I feel like the querying should be done in a model, not in the route. It should be neater and more performant.

**Review by jlong5795 — CHANGES_REQUESTED** [review]

**Comment by jlong5795** [comment]
Slack having issues please join zoom

**Comment by MarkRivera** [comment]
> Slack having issues please join zoom

Can you pass me the link?

**Comment by jlong5795** [comment]
https://lambdaschool.zoom.us/j/7912054792

**Pull request merged**